### DISCUSSION TITLE
When k3s uses an external mysql database, can the database be switched without reinstalling

### DISCUSSION THREAD

**Discussion by 983183947** [k3s-io/k3s#8131] [Q&A] [answered]
When k3s uses an external mysql database, can the database be switched without reinstalling? I want to directly modify the /etc/systemd/system/k3s.service.env file to achieve this situation is feasible?

**Comment by brandond** [comment, marked as answer]
You can change the datastore address sure, but you'd be responsible for migrating the data from one server or database to another yourself.
